packages/cloud_functions/cloud_functions/lib/src/firebase_functions.dart

```diff
@@ -74,13 +85,14 @@ class FirebaseFunctions extends FirebasePluginPlatform {
 
     // Android considers localhost as 10.0.2.2 - automatically handle this for users.
     if (!kIsWeb && defaultTargetPlatform == TargetPlatform.android) {
-      if (origin.startsWith('http://localhost')) {
-        _origin = origin.replaceFirst('http://localhost', 'http://10.0.2.2');
-        return;
-      }
-      if (origin.startsWith('http://127.0.0.1')) {
-        _origin = origin.replaceFirst('http://127.0.0.1', 'http://10.0.2.2');
-        return;
+      for (final localhostOriginHostname in localhostOriginHostnames) {
```

Comment:
I personally prefer the previous code

To me this feels too abstracted.

Reply:
@rrousselGit As you wish, you are the power here, just reverted this changes 👌🏻